xbmc/addons/gui/GUIDialogAddonInfo.cpp

```diff
@@ -329,9 +334,26 @@ void CGUIDialogAddonInfo::OnInstall()
   if (!g_passwordManager.CheckMenuLock(WINDOW_ADDON_BROWSER))
     return;
 
-  if (m_localAddon || !m_item->HasAddonInfo())
+  if (!m_item->HasAddonInfo())
     return;
 
+  if (m_localAddon)
+  {
+    std::string header = g_localizeStrings.Get(19098); // Warning!
```

Comment:
No need for this variable - as only used one time here. if you want to stick with it, make it at list `const`.
Same applies for `text`.